cloud/aws/actuators/machine/actuator.go

```diff
@@ -210,7 +280,7 @@ func (a *Actuator) CreateMachine(cluster *clusterv1.Cluster, machine *clusterv1.
 	}
 	tagList = append(tagList, []*ec2.Tag{
 		{Key: aws.String("clusterid"), Value: aws.String(cluster.Name)},
-		{Key: aws.String("kubernetes.io/cluster/" + cluster.Name), Value: aws.String(cluster.Name)},
+		{Key: aws.String("kubernetes.io/cluster/" + cluster.Name), Value: aws.String("owned")},
```

Comment:
Why `owned`?

Reply:
this matches current installer behaviour